lib/ExtUtils/MM_NW5.pm

```diff
@@ -192,7 +192,7 @@ MAKE_FRAG
     }
     # Reconstruct the X.Y.Z version.
     my $version = join '.', map { sprintf "%d", $_ }
-                              $] =~ /(\d)\.(\d{3})(\d{2})/;
+                              "$]" =~ /(\d)\.(\d{3})(\d{2})/;
```

Comment:
This stringification isn't actually necessary, since it already is used only as a string here.

Reply:
I see this is from #333, that has been merged to master so you should rebase to current master.

Reply:
Done.